packages/toolpad-app/src/runtime/AppThemeProvider.tsx

```diff
@@ -21,8 +23,11 @@ export function createToolpadTheme(toolpadTheme: AppTheme = {}): ThemeOptions {
     palette.mode = mode;
   }
 
+  const designTokens = getDesignTokens(mode || 'light');
+
   return createTheme({
     palette,
+    spacing: designTokens.spacing,
```

Comment:
@apedroferreira Doing a second review of this and I believe this is not the right approach. There are two things here:

1. The editor theme and app theme should be completely isolated from each other. 
2. If I understand correctly, shouldn't the spacing in columns be the same as the `gap` in the `PageRow` that's currently being edited? That means the spacing could potentially change based on which row is being edited and should be read from the actual application.